error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StreamRules.java

```diff
@@ -297,6 +297,18 @@ boolean after(Stream<T> stream) {
     }
   }
 
+  static final class StreamMapToValuesFromMap<K, V> {
```

Comment:
(a) Let's add some light-weight JavaDoc on this class? E.g. similar to `StreamNoneMatch`.
(b) The Refaster rules are most often called after the resulting code, see other examples in this class. In this scenario, this would be `StreamMapFilter`. Can we rename it? :eyes:

Reply:
I added documentation and renamed the rule. The name feels a bit too generic to me though, as it could also apply to other patterns. WDYT?